src/model/sample.rs

```diff
@@ -160,7 +160,7 @@ impl RecordBuffer {
 
 
 /// An observation for or against a variant.
-#[derive(Clone, Debug, RustcDecodable, RustcEncodable)]
```

Comment:
From what I understand, we cannot have a dependency on  a different version of `csv` in our setup, as the compiler complains about `rust-bio` >= `0.15` as:

```
error[E0277]: the trait bound `bio::stats::LogProb: rustc_serialize::Decodable` is not satisfied
   --> src/model/sample.rs:163:24
    |
163 | #[derive(Clone, Debug, RustcDecodable, RustcEncodable)]
    |                        ^^^^^^^^^^^^^^ the trait `rustc_serialize::Decodable` is not implemented for `bio::stats::LogProb`
    |
    = note: required by `rustc_serialize::Decodable::decode`
```

As I understand it, the use of the `LogProb`s in here, which are using the newer `csv` version in `rust-bio` to do their deriving (with `serde`), we cannot use the older `csv` version for deriving Traits, here. Or am I missing something?

Reply:
Ok, makes sense.